src/index.ts

```diff
@@ -136,9 +136,9 @@ function findSymbolSourceFile(symbol: TSSymbol, project: ProjectReflection) {
     const sources = srcMapConverter.toObject().sources as string[]
 
     for (const source of sources) {
-      const srcFileName = path.resolve(srcDirPath, source)
+      const srcFileName = path.resolve(srcDirPath, source).replace(/\\/g, '/')
```

Comment:
**suggestion:** Try to use [`path.normalize()`](https://nodejs.org/dist/latest-v16.x/docs/api/path.html#pathnormalizepath)?

Reply:
done